e2e/pinned_dependencies_test.go

```diff
@@ -49,7 +49,7 @@ var _ = Describe("E2E TEST:"+checks.CheckPinnedDependencies, func() {
 			expected := scut.TestReturn{
 				Errors:        nil,
 				Score:         checker.MinResultScore,
-				NumberOfWarn:  154,
+				NumberOfWarn:  152,
```

Comment:
This is because we are no longer parsing `ksh`: 
https://github.com/ossf-tests/scorecard-check-pinned-dependencies-e2e/blob/91af031ed4e8f5f70d0db2bfd8c126cc27332e1d/.github/workflows/github-workflow-wget-across-steps#L43

Reply:
I've updated the file in https://github.com/ossf-tests/scorecard-check-pinned-dependencies-e2e/blob/main/.github/workflows/github-workflow-wget-across-steps to use `bash` so you need not update this test.